app/code/Magento/Paypal/Model/Payflow/Service/Response/Handler/FraudHandler.php

```diff
@@ -39,16 +40,23 @@ class FraudHandler implements HandlerInterface
      */
     private $xmlSecurity;
 
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
     /**
      * Constructor
      *
      * @param Info $paypalInfoManager
      * @param Security $xmlSecurity
+     * @param LoggerInterface $logger
      */
-    public function __construct(Info $paypalInfoManager, Security $xmlSecurity)
+    public function __construct(Info $paypalInfoManager, Security $xmlSecurity, LoggerInterface $logger)
     {
```

Comment:
This would be easier to read if it were split over multiple lines.
```suggestion
    public function __construct(
        Info $paypalInfoManager,
        Security $xmlSecurity,
        LoggerInterface $logger,
    ) {
```